services/supervisor/src/lib.rs

```diff
@@ -36,15 +36,31 @@ use exonum::{
 use exonum_derive::*;
 use exonum_merkledb::Snapshot;
 
+pub mod mode;
+
 mod api;
 mod configure;
 mod errors;
 mod proto;
 mod proto_structures;
 mod schema;
-pub mod simple;
 mod transactions;
 
+/// Creates a new decentralized `Supervisor`.
+pub fn decentralized_supervisor() -> Supervisor<mode::Decentralized> {
+    Supervisor::<mode::Decentralized>::new()
```

Comment:
I think that `pub type SimpleSupervisor = Supervisor<mode::Simple>` whould be better.